exec/cnex/global.c

```diff
@@ -203,11 +203,11 @@ struct tagTVariables {
 void global_init(int argc, char *argv[], int iArgStart)
 {
     // Init io module
-    VAR_stderr.address = cell_createAddressCell((void*)stderr);
+    VAR_stderr.address = cell_createOtherCell((void*)stderr);
```

Comment:
You do not need `(void*)` casts here. (It's never needed to cast to `(void*)`, stop doing it, all it does is hide bugs.)